File: crushorfriend.cs
using System;

class Program {
    static void Greet(string inputStr) {
        string message = "You have a friend!";

        if (inputStr == "crush") {
            message = "You have a crush!";
        }

        Console.WriteLine(message);
    }

    static void Main() {
        // crush, friend
        Greet("crush");
    }
}

File: crushorfriend.java
public class Program {
    static void greet(String inputStr) {
        String message = "You have a friend!";

        if (inputStr.equals("crush")) {
            message = "You have a crush!";
        }

        System.out.println(message);
    }

    public static void main(String[] args) {
        // crush, friend
        greet("crush");
    }
}
